generators/languages/templates/src/main/webapp/i18n/en/_activate.json

```diff
@@ -3,9 +3,9 @@
         "title": "Activation",
         "messages": {
             <%_ if (clientFramework === 'angular1') { _%>
-            "success": "<strong>Your user has been activated.</strong> Please <a class=\"alert-link\" href=\"\" ng-click=\"vm.login()\">sign in</a>.",
+            "success": "<strong>Your user account has been activated.</strong> Please <a class=\"alert-link\" href=\"\" ng-click=\"vm.login()\">sign in</a>.",
             <%_ } else { _%>
-            "success": "<strong>Your user has been activated.</strong> Please ",
+            "success": "<strong>Your user account has been activated.</strong> Please sign in.",
```

Comment:
Please remove `sign in`, it is set up in the component via `<a class="alert-link" (click)="login()" jhiTranslate="global.messages.info.authenticated.link">sign in</a>`